Makefile

```diff
@@ -4,3 +4,11 @@ run-generators: gen-api
 
 gen-api:
 	oapi-codegen --config codegen/configs/nuts-storage-api-server.yaml codegen/api-specs/nuts-storage-api-v1.yaml | gofmt > api/v1/generated.go
+
+docker:
```

Comment:
Could be ambiguous, but there's no other make targets that could interfer. So I'd like to keep it short, otherwise the `make` command would become longer and longer.

But I can rename it to `build`.